### PR TITLE
Remove broken version based link to source.

### PR DESCRIPTION
**Pull request by leedale1981**
Link to source container version based link that was broken. Propose to use non version based link.

### Proposed changes

Link to source container version based link that was broken. Propose to use non version based link.

### CODE REVIEW

File: storage/storagedriver/select-storage-driver.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ Docker supports the following storage drivers:
    driver is poor, and is not generally recommended for production use.
 
 Docker's source code defines the selection order. You can see the order at
-[the source code for Docker Engine - Community {{ site.docker_ce_version }}](https://github.com/docker/docker-ce/blob/{{ site.docker_ce_version }}/components/engine/daemon/graphdriver/driver_linux.go#L50)
```
Comment (usha-mandya) [inline storage/storagedriver/select-storage-driver.md:58]: @thaJeztah Should this go to https://github.com/moby/moby/blob/20.10/daemon/graphdriver/driver_linux.go#L50 now?

File: storage/storagedriver/select-storage-driver.md [hunk 2]
```diff
@@ -55,7 +55,7 @@ Docker supports the following storage drivers:
...
+[the source code for Docker Engine - Community (https://github.com/docker/docker-ce/blob/master/components/engine/daemon/graphdriver/driver_linux.go#L50)
```
Comment (thaJeztah) [inline storage/storagedriver/select-storage-driver.md:58]: Yes, I think it'll be better to point to the version of the current release; we can remove the ` - Community` as well (it was there to differentiate between "enterprise" and "community"). Lines moved a bit as well, so changed that below to `#L52-L53`;

```suggestion
[the source code for Docker Engine {{ site.docker_ce_version }}](https://github.com/docker/docker-ce/blob/{{ site.docker_ce_version }}/components/engine/daemon/graphdriver/driver_linux.go#L52-L53)
```

**Review by thaJeztah — CHANGES_REQUESTED** [review]

File: storage/storagedriver/select-storage-driver.md [hunk 3]
```diff
@@ -55,7 +55,7 @@ Docker supports the following storage drivers:
    driver is poor, and is not generally recommended for production use.
 
 Docker's source code defines the selection order. You can see the order at
-[the source code for Docker Engine - Community {{ site.docker_ce_version }}](https://github.com/docker/docker-ce/blob/{{ site.docker_ce_version }}/components/engine/daemon/graphdriver/driver_linux.go#L50)
+[the source code for Docker Engine {{ site.docker_ce_version }}](https://github.com/docker/docker-ce/blob/{{ site.docker_ce_version }}/components/engine/daemon/graphdriver/driver_linux.go#L52-L53)
```
Comment (thaJeztah) [inline storage/storagedriver/select-storage-driver.md:58]: whoops, my bad, I should've changed to;

```suggestion
[the source code for Docker Engine {{ site.docker_ce_version }}](https://github.com/moby/moby/blob/{{ site.docker_ce_version }}/daemon/graphdriver/driver_linux.go#L52-L53)
```

**Review by usha-mandya — APPROVED** [review]

**Review by thaJeztah — APPROVED** [review]
LGTM, thanks!

**Pull request merged**